GVFS/GVFS.Mount/InProcessMountVerb.cs

```diff
@@ -129,16 +129,16 @@ private JsonTracer CreateTracer(GVFSEnlistment enlistment, EventLevel verbosity,
             string mountId = null;
 
             GitProcess git = new GitProcess(enlistment);
-            GitProcess.Result configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
-            if (!configResult.HasErrors)
+            GitProcess.ConfigResult configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.EnlistmentId);
+            if (configResult.TryParseAsString(out enlistmentId, out string _))
             {
-                enlistmentId = configResult.Output.Trim();
+                enlistmentId = enlistmentId.Trim();
             }
 
             configResult = git.GetFromLocalConfig(GVFSConstants.GitConfig.MountId);
-            if (!configResult.HasErrors)
+            if (configResult.TryParseAsString(out mountId, out string _))
             {
-                mountId = configResult.Output.Trim();
+                mountId = mountId.Trim();

```

Comment:
Same thing here, `mountId` could be `null`.

Reply:
Fixed by allowing a default parameter, defaulting to string.Empty.